src/main/java/org/folio/dew/batch/acquisitions/edifact/jobs/EdifactExportJobConfig.java

```diff
@@ -21,18 +22,16 @@ public class EdifactExportJobConfig {
   public static final String POL_MEM_KEY = "poLineIds";
 
   private Job constructEdifactExportJob(JobBuilder jobBuilder,
-                                        EdiExportJobCompletionListener ediExportJobCompletionListener,
-                                        Step mapToEdifactOrdersStep,
-                                        Step saveToFTPStep,
-                                        Step saveToMinIOStep,
-                                        Step createExportHistoryRecordsStep) {
-    return jobBuilder.incrementer(new RunIdIncrementer())
-      .listener(ediExportJobCompletionListener)
-      .start(mapToEdifactOrdersStep)
-      .next(saveToMinIOStep)
-      .next(saveToFTPStep)
-      .next(createExportHistoryRecordsStep)
-      .build();
+                                        JobExecutionListener listener,
+                                        Step... steps) {
+    var builder = jobBuilder
+      .incrementer(new RunIdIncrementer())
+      .listener(listener)
+      .start(steps[0]);
+    for (int i = 1; i < steps.length; i++) {
+      builder = builder.next(steps[i]);
```

Comment:
a bit harder to understand the sequence now, need to find where steps are declared in this case

Reply:
Should be reverted to older version now